LayoutTests/accessibility/ax-thread-text-apis/attributed-string-includes-insertion-deletion.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE HTML><!-- webkit-test-runner [ runSingly=true AccessibilityThreadTextApisEnabled=true ] -->
+<!-- This is a modern-style rewrite of existing test mac/mark-role.html. After accessibilityThreadTextApisEnabled is enabled by default, we should replace mac/mark-role.html with this file. -->
```

Comment:
nit: `mac/mark-role.html` should probably be `mac/attributed-string-includes-insertion-deletion.html`

Reply:
Good catch, thanks!